library/init/data/list/lemmas.lean

```diff
@@ -154,6 +158,24 @@ by induction l; simph
 @[simp] lemma map_reverse (f : α → β) (l : list α) : map f (reverse l) = reverse (map f l) :=
 by induction l; simph
 
+lemma reverse_core_append' (xs ys zs : list α) :
```

Comment:
I'm going to check where we need it. Why does Leo not prove anything about reverse_core directly?

I can do without reverse_core_append[']. I'll take it out.